src/utils/ffmpeg/skipSilence.ts

```
@@ -0,0 +1,15 @@
+import { getABRepeatRaw } from '../db/sqlAPI';
```

ESLint

[error] Resolve error: Cannot find module 'babel-plugin-module-resolver'
Require stack:
- /home/jailuser/git/node_modules/eslint-import-resolver-babel-module/lib/index.js
- /home/jailuser/git/node_modules/eslint-module-utils/resolve.js
- /home/jailuser/git/node_modules/eslint-plugin-import/lib/rules/no-unresolved.js
- /home/jailuser/git/node_modules/eslint-plugin-import/lib/index.js
- /home/jailuser/git/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
    at Module._resolveFilename (node:internal/modules/cjs/loader:1405:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1061:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1066:22)
    at Module._load (node:internal/modules/cjs/loader:1215:37)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
    at Module.require (node:internal/modules/cjs/loader:1491:12)
    at require (node:internal/modules/helpers:135:16)
    at Object.<anonymous> (/home/jailuser/git/node_modules/eslint-import-resolver-babel-module/lib/index.js:8:5)
    at Module._compile (node:internal/modules/cjs/loader:1734:14)

(import/namespace)